src/formatters/junit_formatter.js

```diff
@@ -0,0 +1,34 @@
+const escapeHtml = require('escape-html');
+
+export default function JUnitFormatter(errorsGroupedByFile) {
+  const files = Object.keys(errorsGroupedByFile);
+
+  var xml = '<?xml version="1.0" encoding="utf-8"?>\n';
+
+  xml = xml + '<testsuites>\n';
```

Comment:
@mscharley do you know if the `package="..."` attribute is required? If so, I'll add `package="org.graphql-schema-linter"`.

Reply:
I can't speak to whether it is required or not, but it can't hurt to include something like your suggestion can it?